Source/WebCore/platform/mediastream/mac/CoreAudioCaptureDeviceManager.cpp

```diff
@@ -150,8 +155,19 @@ static bool isValidCaptureDevice(const CoreAudioCaptureDevice& device, bool filt
         return false;
     }
 
-    if (device.label().contains("WebexMediaAudioDevice"_s)) {
-        RELEASE_LOG(WebRTC, "Ignoring webex audio device");
+    // FIXME: We might want to use properties like whether a device can be selected as default once we move device enumeration to GPUProcess.
+    if (isVirtualDeviceFromLabel(device.label())) {
+        RELEASE_LOG(WebRTC, "Ignoring virtual audio device '%s'", device.label().utf8().data());
+        return false;
+    }
+
+    return true;
+}
+
+static bool isValidSpeakerDevice(const CoreAudioCaptureDevice& device)
+{
+    if (isVirtualDeviceFromLabel(device.label())) {
+        RELEASE_LOG(WebRTC, "Ignoring virtual audio device");
```

Comment:
Nit: `isVirtualDeviceFromLabel` will have already logged that the device is being ignored

Reply:
Not after the refactoring, but it might be better to make the logging specific to speaker. I'll fix that.